be/src/util/runtime_profile.cpp

```diff
@@ -297,15 +297,15 @@ RuntimeProfile* RuntimeProfile::create_child(const std::string& name, bool inden
 }
 
 void RuntimeProfile::add_child_unlock(RuntimeProfile* child, bool indent, RuntimeProfile* loc) {
-    DCHECK(child != NULL);
+    DCHECK(child != nullptr);
     _child_map[child->_name] = child;
 
-    if (loc == NULL) {
+    if (loc == nullptr) {
         _children.push_back(std::make_pair(child, indent));
     } else {
         for (ChildVector::iterator it = _children.begin(); it != _children.end(); ++it) {
             if (it->first == loc) {
-                _children.insert(++it, std::make_pair(child, indent));
+                _children.insert(it++, std::make_pair(child, indent));
```

Comment:
Seems no need to plus? because we return directly.

Reply:
Right! I have Fixed It